.github/pull_request_template.md

```diff
@@ -1,26 +1,17 @@
-## Pull Request Info
+# Pull Request Info
 
-[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)
-
-### Issue JIRA link:
-https://jira.mongodb.org/browse/DOCSP-NNNNN
+[JIRA](https://jira.mongodb.org)
+[Staging](https://www.example.com)
```

Comment:
Suggestion: keep the JIRA and staging link placeholders as we decided in our team sync that those were ok. We decided we aren't going to recommend community contributions, so these don't have to be in a form that the community understands.